src/renderer/src/personal-views/activity-view/activity-asset.ts

```diff
@@ -116,7 +117,7 @@ export class ActivityAsset extends LitElement {
       case 'pending':
         return html`<div>Loading...</div>`;
       case 'complete':
-        return html`${this.appletName.value.value}`;
```

Comment:
If this function is only used to get the string value, not an html template string then the `html` part should also removed from the `'pending'` and `'error'` cases I think

Reply:
Right now it's a mixture between html and raw string

Reply:
And if it's not used for rendering, maybe the function could just be renamed to `appletName()` instead of `renderAppletName()` but that's a detail.

Reply:
removed html tag and changed to getAppletName